B2B Add payment limitation info around purchase orders (#976)
magento/merchdocs: src/customers/account-dashboard-approval-rules.md

_Company Approval Rules_

Depending on the company role and permissions, users may have the option to edit, delete, or view approval rules.

> ## Purchase order payment methods

Reviewer: Need a blank before the heading

```suggestion

## Purchase order payment methods
```

Purchase order approval supports both online and offline payment methods.
All default offline payment methods are supported for purchase order approvals.

For online payments, the following methods are supported:

* Paypal Express
* Braintree payments

> {:.bs-callout-warning}

Reviewer: ```suggestion
{:.bs-callout-info}
```

Purchase orders should use an "In-Context Checkout Experience". Out-of-Context checkouts are not supported as they bypass the normal checkout flow.
Generally, "In-Context" means you stay on the Magento site to complete the process. "Out-of-Context" is when you are taken to another site to complete the purchase.

> Also, using gift cards, store credit, or reward points with online payment methods for purchase orders is not supported. Enabling these features with online payments may cause some unexpected behavior. It is recommended to disable gift cards, store credit, and reward points when online payments are enabled for purchase orders.

Reviewer: ```suggestion

{:.bs-callout-warning}
Using gift cards, store credit, or reward points with online payment methods for purchase orders is not supported. Enabling these features with online payments may cause some unexpected behavior. It is recommended to disable gift cards, store credit, and reward points when online payments are enabled for purchase orders.
```

Depending on the company role and permissions, users may have the option to edit, delete, or view approval rules.

## Purchase order payment methods

Purchase order approval supports both online and offline payment methods.
All default offline payment methods are supported for purchase order approvals.

For online payments, the following methods are supported:

* Paypal Express
* Braintree payments

{:.bs-callout-info}
Purchase orders should use an _In-Context_ checkout experience. _Out-of-Context_ checkouts are not supported because they bypass the normal checkout flow.

> Generally, _In-Context_ means the customer stays on the Magento site to complete the process. _Out-of-Context" is when the customer is taken to another site to complete the purchase.

Reviewer: My boo boo from last round

```suggestion
Generally, _In-Context_ means the customer stays on the Magento site to complete the process. _Out-of-Context_ is when the customer is taken to another site to complete the purchase.
```
